pkg/controllers/repositoryserver/utils.go

```diff
@@ -264,10 +264,17 @@ func getVolumes(ctx context.Context, cli kubernetes.Interface, secret *corev1.Se
 			return nil, errors.New("Claim name not set for file store location secret, failed to retrieve PVC")
 		}
 		claimNameString := string(claimName)
+		pvc, err := cli.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, claimNameString, metav1.GetOptions{})
+		if err != nil {
+			return nil, errors.Wrapf(err, "Failed to validate if PVC %s:%s exists", namespace, claimName)
+		}
```

Comment:
it looks like we are making the same GET PVC call twice here, is that intentional?

Reply:
fixed